src/Kernel32.Tests.Shared/NTStatusExceptionFacts.cs

```diff
@@ -54,10 +58,11 @@ public void NTStatusException_Informational_Message()
     }
 
     [Fact]
+    [UseCulture("en-US")]
```

Comment:
Why do we need to both set the culture here, and also pass in `0x409` as parameters inside the test itself? Which mechanism of setting the language are we testing?